scalafmt-tests/src/test/scala/org/scalafmt/util/DeleteTree.scala

```diff
@@ -0,0 +1,44 @@
+package org.scalafmt.util
+
+import java.io.IOException
+import java.nio.file.{FileVisitResult, FileVisitor, Files, Path}
+import java.nio.file.attribute.BasicFileAttributes
+
+object DeleteTree {
```

Comment:
If this class used in tests only, it's reasonable to move it into `src/test/scala/....` directory

Reply:
It looks like it’s in the test directory to me. Do you mean somewhere other than _scalafmt-tests/src/test/scala_?

Reply:
Oh, sorry, just my mistake